File: WebSite/EulerAList/pset15.cpp
#include <iostream>

using namespace std;

int main()
{
    long arr[21][21];
    for (int i = 0; i < 21; i ++)
    {
        arr[i][0] = 1;
        arr[0][i] = 1;
    }
    for (int i = 1; i < 21; i ++)
    {
        for (int j = 1; j < 21; j ++)
        {
            arr[i][j] = arr[i - 1][j] + arr[i][j - 1];
        }
    }
    cout << arr[20][20] << endl;
}

File: WebSite/EulerAList/pset15.py
arr = [[1 for x in range(21)] for y in range(21)]
for i in range(1, 21):
    for j in range(1, 21):
        arr[i][j] = arr[i - 1][j] + arr[i][j - 1]
print(arr[20][20])